tests/data/monopoly.csv

```diff
@@ -1,23 +1,23 @@
-name,rent,house_1,house_2,house_3,house_4,hotel,deed_cost,house_cost,color,tile
-Mediterranean Avenue,2,10,30,90,160,250,60,50,purple,1
-Baltic Avenue,4,20,60,180,320,450,60,50,purple,3
-Oriental Avenue,6,30,90,270,400,550,100,50,light_blue,6
-Vermont Avenue,6,30,90,270,400,550,100,50,light_blue,8
-Connecticut Avenue,8,40,100,300,450,600,120,50,light_blue,9
-St. Charles Place,10,50,150,450,625,750,140,100,pink,11
-States Avenue,10,50,150,450,625,750,140,100,pink,13
-Virginia Avenue,12,60,180,500,700,900,160,100,pink,14
-Tennessee Avenue,14,70,200,550,750,950,180,100,orange,16
-St. James Place,14,70,200,550,750,950,180,100,orange,18
-New York Avenue,16,80,220,600,800,1000,200,100,orange,19
-Kentucky Avenue,18,90,250,700,875,1050,220,150,red,21
-Indiana Avenue,18,90,250,700,875,1050,220,150,red,23
-Illinois Avenue,20,100,300,750,925,1100,240,150,red,24
-Atlantic Avenue,22,110,330,800,975,1150,260,150,yellow,26
-Ventnor Avenue,22,110,330,800,975,1150,260,150,yellow,27
-Marvin Gardens,24,120,360,850,1025,1200,280,150,yellow,29
-Pacific Avenue,26,130,390,900,1100,1275,300,200,green,31
-North Carolina Avenue,26,130,390,900,1100,1275,300,200,green,32
-Pennsylvania Avenue,28,150,450,1000,1200,1400,320,200,green,34
-Park Place,35,175,500,1100,1300,1500,350,200,blue,37
+name,rent,house_1,house_2,house_3,house_4,hotel,deed_cost,house_cost,color,tile
```

Comment:
It feels like something went wrong in git here. Somehow extra spaces got added at the end maybe?

Reply:
It could be. I didn't even need to use this file but Git did something funky. It removed pokemon.json file during the push and the tests failed so I had to re-add it.

Reply:
So is this resolved?

Reply:
If you're having trouble reverting this file in git then I can live with it but it is a strange edit.